components/tabs/stories/tabs-horizontal-compact.stories.js

```diff
@@ -29,29 +28,60 @@ export default {
   }
 };
 
-export const Default = Template.bind({});
-Default.args = horizontalProps;
-
-export const WithIcon = Template.bind({});
-WithIcon.args = horizontalWithIconsProps;
-
-export const IconOnly = Template.bind({});
-IconOnly.args = horizontalIconOnlyProps;
-
-export const CompactEmphasized = Template.bind({});
-CompactEmphasized.args = {
-  isEmphasized: true,
-  ...horizontalProps
+const TabsGroup = ({
+	...args
+}) => {
+	return html`
+		<div style="display: flex; flex-direction: column; gap: 2rem;">
+			${Template({
+          selectorStyle: {"width": "35px"},
```

Comment:
I'm seeing some weirdness when toggling controls that I think is caused by this static selectorStyle. For example, if I turn on icons, the selector indicator looks too short:

![Screen Shot 2024-02-07 at 2 40 36 PM](https://github.com/adobe/spectrum-css/assets/23404383/51dd39cb-afef-4c33-a29c-5df6ce90d9dc)

Reply:
Addressed this!